apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue

```diff
@@ -81,7 +81,7 @@
                     >
                         <p-tooltip v-if="jobStatus.status === JOB_STATE.SUCCESS"
                                    class="icon-fill-wrapper success"
-                                   :contents="$t('INVENTORY.COLLECTOR.MAIN.JOB_SUCCESS', {date: 'yyyy-mm-dd hh:mm:ss'})"
+                                   :contents="$t('INVENTORY.COLLECTOR.MAIN.JOB_SUCCESS', {date: dayjs(jobStatus.finished_at).format('YYYY-MM-DD hh:mm:ss')})"
                                    position="top"
```

Comment:
issue: I think this must be converted to user timezone!

Reply:
🥸
dayjs.utc(jobStatus.finished_at).tz(store.state.user.timezone).format('YYYY-MM-DD hh:mm:ss')